index.html

```diff
@@ -160,5 +160,21 @@ <h3 class="pcol">LET’S CONNECT</h3>
                 </ul>
             </div>
            </div>
+           <section class="contact">
+            <div class="text-div">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing,
+                     a feature that you need built or a project that needs coding.
+                      I’d love to help with it </p>
+            </div>
+            <div class="form-div">
+<form class="form1" action="https://formspree.io/f/xpzewknr" method="post">
+    <input class="design" type="text" name="fullname" id="fullname" size="30" placeholder="Enter your Name here" required><br>
```

Comment:
- [ ] You did an amazing job creating all the inputs inside your `<form>` element :+1: :1st_place_medal:, but as a project requirement name field should only accept up to 30 chars. I have noticed that you used `size="30"` to do that but this is not working so I think it would be better if you use `maxlength="30"` attribute for the requirement :+1:

Reply:
Suggested changes made and maxlength attribute added in the codebase.